snap/snapcraft.yaml

```diff
@@ -298,7 +299,15 @@ parts:
         | xargs ./configure
 
       # Copy QEMU configurations (Kconfigs)
-      cp -a ${kata_dir}/tools/packaging/qemu/default-configs/* default-configs/devices/
+      case "$(branch)" in
+      "v5.1.0")
+        cp -a ${kata_dir}/tools/packaging/qemu/default-configs/* default-configs
+        ;;
+
+      "v5.2.0")
```

Comment:
Can we make this `*` for easier QEMU upgrades?

Reply:
Hello @Jakob-Naucke -, I think it is necessary to change the behavior based on qemu version as the location of "default-configs" varies with the it.

Reply:
I see that, but wouldn't it be the same from 5.2 on out, i.e. we could have the special case for 5.1, and then the general case for everything else? Otherwise, this needs updating every time we update QEMU.

Reply:
So, what about changing "5.2.0" to "*" here?

Reply:
exactly, that was my suggestion :smile: